samples/drivers/led_apa102c_bitbang/src/main.c

```diff
@@ -55,11 +55,11 @@ void send_rgb(struct device *gpio_dev, u32_t rgb)
 
 	for (i = 0; i < 32; i++) {
 		/* MSB goes in first */
-		gpio_pin_write(gpio_dev, GPIO_DATA_PIN, !!(rgb & 0x80000000));
+		gpio_pin_set_raw(gpio_dev, GPIO_DATA_PIN, !!(rgb & 0x80000000));
```

Comment:
MISRA-inclined folks would prefer this to change to `(rgb & 0x80000000U) != 0` or (better) `(rgb & BIT(31)) != 0`.

Reply:
Why not, since we are touching this file. Fixed, thanks.